src/main/java/Duke.java

```diff
@@ -1,10 +1,224 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
```

Comment:
You could consider adding javadocs and descriptive headers to your public classes and methods.